### PR TITLE
Add keynote speakers

### LINKED ISSUE #90: Add Keynote speakers to the website

### PR DESCRIPTION
**Pull request by ananyo2012**
Fixes #90

Screenshot:
speaker-snip

### CODE REVIEW

**Comment by ananyo2012** [comment]
Added Carol Willings image as well.
Screenshot:
image

**Comment by vipulgupta2048** [comment]
Umm .. I feel Armin's image is a bit compressed. A little more width to it would look better. Or are the placeholders are of definite size

**Comment by ananyo2012** [comment]
@vipulgupta2048 Here is the updated screenshot
image

**Comment by vipulgupta2048** [comment]
My eyes could be fooling me, I am getting old :see_no_evil: 
But I am not able to make out any difference, I think this is the same screenshot. I see the file changes out have made.  
I would request @pythonindia/pycon-2018-team to review again.

File: index.html [hunk 1]
```diff
@@ -173,32 +173,40 @@ <h1 class="pycon-journey">JOURNEY</h1>
...
                       <h1 class="pycon-speaker">SPEAKERS</h1>
                         <div class="row pycon2018_spe">
-                          <div class="col-md-4 text-center">
-                              <img src="img/border1.png" class="border1" alt="border">
-                             <img src="img/speaker.png" alt="image">
-                             <h3 class="text-right">Robert Downey Jr</h3>
-                             <h5 class="text-right">Google Inc</h5>
+                          <div class="col-md-3 col-md-offset-3 text-center">
+                             <img src="img/border1.png" class="border1" alt="border">
+                             <img src="img/armin-ronacher.png" alt="image">
```
Comment (vipulgupta2048) [inline index.html]: You can add more width from here inline, it the dimensions are not fixed.

**Comment by ananyo2012** [comment]
Here is the screenshot with all the keynote speakers 
image

**Comment by ananyo2012** [comment]
@pythonindia/pycon-2018-team Is this good to go ?

**Comment by vipulgupta2048** [comment]
@ananyo2012 Merging it in.

**Comment by ananyo2012** [comment]
Sorry I forgot to update the PR with the commit for adding Travis Oliphant. I will make a separate PR with the hotfix.

**Pull request merged**